v2/e2e/controller_test.go

```diff
@@ -12,7 +12,7 @@ import (
 
 var _ = Describe("coil-controller", func() {
 	It("should elect a leader instance of coil-controller", func() {
-		kubectlSafe(nil, "-n", "kube-system", "get", "configmaps", "coil-leader")
+		kubectlSafe(nil, "-n", "kube-system", "get", "leases", "coil-leader")
```

Comment:
From controller-runtime v0.12.0, a Lease object is used for the leader election.
ConfigMaps will no longer be created.

ref: https://github.com/kubernetes-sigs/controller-runtime/releases/tag/v0.12.0